### PR TITLE
Vertical Tab for pages

### PR DESCRIPTION
**Pull request by MichelePezza**
I like your addon but toolbar doesn't manage scrolling so number of tabs is limited by screen size.

What I made:
- new mpage.xhtml with two vertical scrollable div: toolbar and page
- modified view.js to click on 'li' instead of 'a' element
- modified dd.js to work with new vertical layout
- removed stickyheader pref

I tested it on my win10 desktop and laptop... I didn't get errors
Probably the changes messed up the code for android device (Is mPage installable on new firefox for android?)

Ciao
MP

### CODE REVIEW

**Comment by mpod** [comment]
Hi Michelle,
thank you for making the pull request. It is an interesting change. Few things:
- sticky header was requested by one of the early users and there are some good reasons for having it, so I would like to leave it.
- vertical tabs cannot be default and the only way for formatting the headers. I use mPage basically as a home page and I never switch between the pages, so I don't need vertical tabs. I understand that they may be useful to you. Can you make a configuration parameter for choosing between vertical and horizontal tabs?

Basically I'm ok with merging the pull request but in that case I will make changes explained above before uploading the new version of add-on to mozilla. Let me know what do you prefer. 

Thank you very much for your contribution.

**Comment by MichelePezza** [comment]
Hi Mpod

I agree with your decision. 

Ciao
M

p.s. sticky header was requested by me

**Comment by mpod** [comment]
Ok, I'll merge the pull request when I find some time to make all other changes. It will take me a week or two. Thanks...

**Comment by mpod** [comment]
Hi Michele,
I merged your changes and added vertical tabs to be optional menu layout. Can you please test it before I create new add-on version? Is sidebar wide enough for your purposes?

Matija

**Comment by MichelePezza** [comment]
Hi Matija

little bit wider will be better.
Is it the definitive layout?
I think that a sidebar without its own scroll lack of usability
Immagine 2021-09-26 191151
Could you consider to add an indipendent scroll for sidebar?

Thx 
Michele

**Comment by mpod** [comment]
I see, will add independent scroll for sidebar, and will also make it wider. 
What do you mean by definitive layout? I'm still open for suggestions (or pull requests).

**Comment by MichelePezza** [comment]
I was talking of a definitive layout to introduce suggestion of an indipendent scroll for sidebar
Thx for considering it
Ciao
Michele
